### ISSUE TITLE
How with Grad-CAM in my own model

### ISSUE THREAD

**Issue by yyssmm** [kazuto1011/grad-cam-pytorch#24]
Hi,
  Thank you for sharing your work! I meet a problem when use Grad-CAM with my own model that have no FC classifier layer and no softmax layer. Would you please help me how to deal with it? Thank you very much!

**Comment by kazuto1011** [comment]
1. **Define the score for which you want to investigate the causality**. In the original Grad-CAM, the score is arbitrary class logit before softmax is applied.
2. Extract the intermediate feature maps in the forward propagation.
3. Compute gradients of the score with respect to the feature maps.
4. Weight the feature maps by the gradient magnitudes.
